src/plugins/graph/graph-types.ts

```diff
@@ -111,3 +113,9 @@ export const kGraphPortalClass = ".canvas-area";
 
 export const GraphEditModes = ["none", "edit", "add"];
 export type GraphEditMode = typeof GraphEditModes[number];
+
+export interface ILocationSetterContext {
+  set: (id: string, location: Point|undefined, size: RectSize|undefined) => void;
+}
+
+export const LocationSetterContext = createContext<ILocationSetterContext|undefined>(undefined);
```

Comment:
My inclination would be to move this definition into its own file since there are a lot of clients of this module and very few clients of the `LocationSetterContext`.